config/charmap.json

```diff
@@ -352,6 +352,43 @@
   "ә": "ae",
   "Ө": "OE",
   "ө": "oe",
+  "Ա": "A",
+  "Բ": "B",
+  "Գ": "G",
+  "Դ": "D",
+  "Ե": "E",
+  "Զ": "Z",
+  "Է": "E'",
+  "Ը": "Y'",
+  "Թ": "T'",
+  "Ժ": "JH",
```

Comment:
Wait, are these correct, the quote is going to be removed anyway?

Reply:
Looking at https://en.wikipedia.org/wiki/Romanization_of_Armenian, there doesn't appear to be a single system that eliminates diacritics and apostrophes/single-quotes, so we may have to get a little creative here.

Reply:
Google Translate seems to have slightly different opinion about it, event though quotes are still present in some cases:

```
Է Ը Թ Ծ Ձ Ռ Փ Ք Օ

E Y T’ TS DZ RR P’ K’ O - google translate

E' Y' T' C' D' R' P' Q' O'' - slugify PR
```

Reply:
Do we want to push a release with this along with the `trim` option, or should we revert this and re-land it once this stuff is cleared up? (I have no opinion.)

Reply:
I'm for reverting it, my mistake for merging it too quickly. It doesn't have tests and that's the reason why I missed that issue.

Reply:
I used this site as a reference: https://translit.cc/am/